docs/jwst/pipeline/calwebb_spec2.rst

```diff
@@ -243,8 +243,14 @@ The input to the ``Spec2Pipeline`` pipeline is a countrate exposure, in the form
 of either "_rate" or "_rateints" data. A single input file can be processed or an
 ASN file listing multiple inputs can be used, in which case the processing steps
 will be applied to each input exposure, one at a time.
-If "_rateints" products are used as input, each step applies its algorithm to each
-integration in the exposure, where appropriate.
+
+If "_rateints" products are used as input, for modes other than NIRSpec Fixed Slit,
+each step applies its algorithm to each integration in the exposure, where appropriate.
+For the NIRSpec Fixed Slit mode, although the :ref:`flat_field <flat_field_step>`
```

Comment:
You don't need to single out the flat_field step here, because it's not unique in terms of handling 3D data. This PR makes it equal to all the other spec2 steps which were already capable of handling 3D data. So I think this middle phrase should be dropped and just have it say "For the NIRSpec Fixed Slit mode the calwebb_spec2 pipeline will ..."

Reply:
Done